cypress/e2e/firstTests.cy.js

```diff
@@ -0,0 +1,65 @@
+/// <reference types="cypress" />
+const { generateUser } = require('../support/generate');
+
+describe('User should be able', () => {
+
+  it('to sign in with valid data', () => {
+  
+  const { email, password, username } = generateUser();
+
+  cy.request('POST', 'https://api.realworld.io/api/users', {
+    "user": {
+      username,
+      email,
+      password
+    }
+  });
+
+  cy.visit('/');
+  
+  cy.contains('a', 'Sign in')
+    .click();
+
+  cy.get('[placeholder="Email"]')
+    .type(email);
+
+  cy.get('[placeholder="Password"]')
+    .type(password);
+
+  cy.contains('.btn', 'Sign in')
+    .click();
+
+  cy.contains('.nav-link', `${username}`)
```

Comment:
Make it more simple, you don't need interpolation
```suggestion
  cy.contains('.nav-link', username)
```